src/cljs/athens/devcards/athena.cljs

```diff
@@ -262,18 +262,32 @@
 
 
 (defn results-el
-  []
-  [:div (use-style results-heading-style)
-   [:h5 "Results"]
-   [:span (use-style hint-style)
-    "Press "
-    [:kbd "shift + enter"]
-    " to open in right sidebar."]])
+  [state]
+  (let [recent-item-length 40
+        search-term? (str/blank? (:query @state))]
```

Comment:
Why rename this to `search-term?` Please stop renaming keys when it doesn't add value. Just call it `query?`